support/camel-k-maven-plugin/src/main/java/org/apache/camel/k/tooling/maven/GenerateCatalogMojo.java

```diff
@@ -240,6 +240,19 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                         .build()
                 );
             }
+            if (capabilitiesExclusionList != null && !capabilitiesExclusionList.contains("resume")) {
+                runtimeSpec.putCapability(
+                        "resume",
+                        CamelCapability.forArtifact(
+                                "org.apache.camel.k", "camel-k-resume"));
+
+                catalogSpec.putArtifact(
+                        new CamelArtifact.Builder()
+                                .groupId("org.apache.camel.k")
+                                .artifactId("camel-k-resume")
```

Comment:
we should probably be able to have contextualization, like camel-k-resume-kafka, camel-k-resume-jdbc and have a way to include the specific technology.